ipatests/test_integration/test_epn.py

```diff
@@ -487,3 +487,23 @@ def test_EPN_delay_config(self, cleanupmail):
         self.master.put_file_contents('/etc/ipa/epn.conf', epn_conf)
         result = tasks.ipa_epn(self.master, raiseonerr=False)
         assert "smtp_delay cannot be less than zero" in result.stderr_text
+
+    def test_EPN_admin(self):
+        """The admin user is special and has no givenName by default
+           It also doesn't by default have an e-mail address
+           Check --dry-run output.
+        """
+        epn_conf = textwrap.dedent('''
+            [global]
+        ''')
+        self.master.put_file_contents('/etc/ipa/epn.conf', epn_conf)
+        self.master.run_command(
+            ['ipa', 'user-mod', 'admin', '--password-expiration',
+             datetime_to_generalized_time(
+                 datetime.datetime.utcnow() + datetime.timedelta(days=7)
+             )]
+        )
+        (unused, stderr_text) = self._check_epn_output(
```

Comment:
@rcritten self._check_epn_output now returns a three-item tuple. Can you add "_unused" as the third member of the tuple please?

Reply:
Solved, thanks